### ISSUE TITLE
0.54 and blob support introduced breaking changes for fetch

### ISSUE THREAD

**Issue by ksegla** [facebook/react-native#18324]
I was on 0.48.4 and used to call fetch in order to get page previews (title, description, image, etc.). 
I just upgraded to 0.54 and my code (see below) does not work anymore. _bodyInit now only contains information about the blob. 
```
fetch(link)
  .then((response) => {
    // I would then use response._bodyInit to get and treat page content
```
This may be the result of this commit: [be56a3e ](https://github.com/facebook/react-native/commit/be56a3e)

### Environment

Environment:
  OS: Windows 10
  Node: 6.11.4
  Yarn: Not Found
  npm: 5.7.1
  Watchman: Not Found
  Xcode: N/A
  Android Studio: Not Found [Weird, I do have Android Studio installed and it was running]

Packages: (wanted => installed)
  react: 16.3.0-alpha.1 => 16.3.0-alpha.1
  react-native: 0.54.0 => 0.54.0

### Expected Behavior
_bodyInit from fetch response has the content of the page

### Actual Behavior
_bodyInit from fetch response only has info about the blob (size etc.)

### Steps to Reproduce
Basically the example from https://facebook.github.io/react-native/docs/network.html#using-fetch
```
async function getMoviesFromApi() {
  try {
    let response = await fetch(
      'https://facebook.github.io/react-native/movies.json'
    );
   alert(JSON.stringify(response)); //used to have a key _bodyInit with all the content
  } catch (error) {
    console.error(error);
  }
}
```

**Comment by ksegla** [comment]
Well, I should have been using response.text(). Closing this.

**Comment by pencilcheck** [comment]
Found a way: (reference: https://medium.com/programmers-developers/convert-blob-to-string-in-javascript-944c15ad7d52)
```
  const reader = new FileReader();
  // Start reading the blob as text.
  reader.readAsText(response._bodyInit);
  // This fires after the blob has been read/loaded.
  var result = await new Promise((resolve, reject) => {
    reader.addEventListener('loadend', (e) => {
      const text = e.currentTarget._result;
      resolve(text);
    });
  })
  var json = JSON.parse(result)

```
